assets/components/contributionsThankYou/contributionsThankYouPage.jsx

```diff
@@ -95,4 +121,4 @@ function RegularContributionsThankYouPage(props: PropTypes) {
 
 // ----- Exports ----- //
 
-export default connect(mapStateToProps, mapDispatchToProps)(RegularContributionsThankYouPage);
+export default connect(mapStateToProps, mapDispatchToProps)(ContributionsThankYouPage);
```

Comment:
`connect`ing this to the Redux store ties it to the state on a particular page. Reusable components can be connected to the Redux store on specific pages using [container components](https://github.com/guardian/support-frontend/wiki/Client-side-Architecture#presentational-and-container-components).